pyfar/plot/_line.py

```diff
@@ -152,19 +152,34 @@ def _time(signal, dB=False, log_prefix=20, log_reference=1,
         ymin = ymax - 90
         ymax = ymax + 10
 
+    # auto detect the time unit
+    if unit is None:
+        unit = _time_auto_unit(signal.times[..., -1])
+    # set the unit
+    if unit == 's':
+        times = signal.times.T
+    elif unit == 'ms':
+        times = signal.times.T * 1e3
+    elif unit == 'mus':
+        times = signal.times.T * 1e6
+        unit = 'micro s'
```

Comment:
I would prefer to use the unicode micro sign µ here, or do you expect errors with the label fonts?

Reply:
I did not find a simple uniode solution and used Matplotlibs build in mathsupport. To make everything look nice I changed the default fonts and font size in our plot styles (the new fonts are slightly larger than the old one, thus the decreased font size).